src/components/BackToTop.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+const path = process.env.PUBLIC_URL;
+const image = '_new/img/buttons/VectorUp.svg';
+
+export const BackToTop: React.FC = () => {
+  const handleScroll = () => {
+    window.scrollTo({
+      top: 0,
+      behavior: 'smooth',
+    });
+  };
+
+  return (
+    <div className="back-to-top">
+      <div className="small_text">Back to top</div>
+      <button
+        className="back-to-top__button"
+        type="button"
+        onClick={() => handleScroll()}
```

Comment:
No need in additional callback
```suggestion
        onClick={handleScroll}
```